src/nbsap/settings.py.docker

```diff
@@ -0,0 +1,256 @@
+WSGI_APPLICATION = 'nbsap.wsgi.application'
+
+TEMPLATE_DIRS = (
+    # Put strings here, like "/home/html/django_templates" or "C:/www/django/templates".
+    # Always use forward slashes, even on Windows.
+    # Don't forget to use absolute paths, not relative paths.
+    os.path.join(BASE_DIR, 'nbsap', 'templates'),
+)
+
+INSTALLED_APPS = (
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.sites',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'django.contrib.admin',
+    'django_assets',
+    'django_webtest',
+    'widget_tweaks',
+    'transmeta',
+    'django.contrib.markup',
+    'tinymce',
+    'nbsap',
+    'chosen',
+    'raven.contrib.django',
+    'gunicorn',
+    'rosetta',
+    # 'south'
+)
+
+# A sample logging configuration. The only tangible logging
+# performed by this configuration is to send an email to
+# the site admins on every HTTP 500 error when DEBUG=False.
+# See http://docs.djangoproject.com/en/dev/topics/logging for
+# more details on how to customize your logging configuration.
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'simple': {
+            'format': '%(levelname)s %(message)s'
+        },
+    },
+    'filters': {
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse'
+        }
+    },
+    'handlers': {
+        'mail_admins': {
+            'level': 'ERROR',
+            'filters': ['require_debug_false'],
+            'class': 'django.utils.log.AdminEmailHandler'
+        },
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+            'formatter': 'simple'
+        },
+    },
+    'loggers': {
+        'django.request': {
+            'handlers': ['mail_admins'],
+            'level': 'ERROR',
+            'propagate': True,
+        },
+        'sentry.errors': {
+            'handlers': ['console'],
+            'level': 'INFO',
+            'propagate': True,
+        },
+    }
+}
+
+# NBSAP special variables
+LOGIN_REDIRECT_URL = '/'
+
+TINYMCE_DEFAULT_CONFIG = {
+    "theme_advanced_buttons1": ("formatselect,"
+                                "separator, bold, italic, "
+                                "underline, strikethrough, separator,"
+                                "justifyleft,justifycenter, justifyright,"
+                                "justifyfull, separator, bullist, numlist,"
+                                "separator, link, code")
+}
+TINYMCE_SPELLCHECKER = False
+TINYMCE_COMPRESSOR = False
+TINYMCE_JS_URL = '/static/js/tiny_mce/tiny_mce.js'
+TINYMCE_JS_ROOT = '/static/js/tiny_mce'
+
+CSS_ASSETS = ()
+
+ASSETS_ROOT = os.path.join(BASE_DIR, 'static')
+
+# Main config
+EU_STRATEGY = False
+EU_STRATEGY_ADD = False
+NAT_STRATEGY = True
+SITE_HEADER = 'NBSAP'
+INFO_HEADER = False
+LAYOUT_FOOTER_LOGO_VISIBLE = False
+LAYOUT_HEADER_LOGO_VISIBLE = False
```

Comment:
Please take EU_STRATEGY, EU_STRATEGY_ADD, NAT_STRATEGY, SITE_HEADER, INFO_HEADER from the environment.